src/main/scala/edu/berkeley/ce/rockslicing/Block.scala

```diff
@@ -115,19 +120,22 @@ object Block {
     // Shift from Block local coordinates to global coordinates
     ((center(0) + centerX, center(1) + centerY, center(2) + centerZ), radius)
   }
-}
 
-/**
-  * A rock block.
-  * @constructor Create a new rock block
-  * @param center Cartesian coordinates for the center of the rock block. The individual
-  * components can be accessed as 'centerX', 'centerY', and 'centerZ'.
-  * @param faces: The faces that define the boundaries of the rock block.
-  */
-case class Block(center: (Double,Double,Double), faces: Seq[Face]) {
-  val (centerX, centerY, centerZ) = center
-  val ((sphereCenterX, sphereCenterY, sphereCenterZ), sphereRadius) =
-      Block.findBoundingSphere(centerX, centerY, centerZ, faces)
+  private def findMaxInscribableRadius: Double = {
+    /*
+     * Equation (18) in Boon et. al., 2015 expresses this as a minimization.
+     * We use a maximization LP to deal with negative distances explicitly
+     * rather than falling back on absolute value.
+     */
```

Comment:
This comment is good; will definitely be helpful is someone is trying to figure out we diverge from Boon's implementation here.